modules/custom/activity_creator/activity.page.inc

```diff
@@ -122,6 +126,12 @@ function template_preprocess_activity(array &$variables) {
   switch ($status) {
     case ACTIVITY_STATUS_RECEIVED:
       $status_class = 'bg-gray-lightest';
+
```

Comment:
@OH17L Please add some comment with the explanation why do we need max-age 0 here.

Reply:
Done.